encoding.bs

```diff
@@ -1923,8 +1923,7 @@ consumers of content generated with <a>GBK</a>'s <a for=/>encoder</a>.
 
    <li><p>Unset the <a>EUC-JP jis0212 flag</a>.
 
-   <li><p>If <var>byte</var> is not in the range 0xA1 to 0xFE, inclusive,
-   <a>prepend</a> <var>byte</var> to
+   <li><p>If <var>byte</var> is an <a>ASCII byte</a>, <a>prepend</a> <var>byte</var> to
```

Comment:
If we only change null to non-null checks when it would affect at least two null checks, we'd have to change gb18030, Big5, EUC-JP (though per above it doesn't really need a null check), Shift_JIS, and EUC-KR.

I'm okay either way.